### PR TITLE
Chart changes for Connection token handler acceptance tests

### PR DESCRIPTION
**Pull request by Maladie**
**Description**

Changes proposed in this pull request:

- Role, RoleBinding and test pod for acceptance tests

**Related issue(s)**

#2910

### CODE REVIEW

File: resources/application-connector/charts/connection-token-handler/templates/tests/role-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-connection-token-handler-tests
```
Comment (franpog859) [inline resources/application-connector/charts/connection-token-handler/templates/tests/role-binding.yaml]: The same thing I mentioned in `test-acceptance.yaml` file. For consistency I suggest to use Chart values everywhere it is possible

File: resources/application-connector/charts/connection-token-handler/templates/tests/test-acceptance.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  labels:
+    helm-chart-test: "true"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  serviceAccountName: {{ .Release.Name }}-connection-token-handler-tests
+  containers:
+  - name: {{ .Release.Name }}-connection-token-handler-tests
+    image: {{ .Values.global.containerRegistry.path }}/{{ .Values.global.connection_token_handler_tests.dir }}connection-token-handler-tests:{{ .Values.global.connection_token_handler_tests.version }}
+    imagePullPolicy: IfNotPresent
```
[the remark this answers is not in the document]
Comment (Maladie) [reply resources/application-connector/charts/connection-token-handler/templates/tests/test-acceptance.yaml]: Done

File: resources/application-connector/charts/connection-token-handler/templates/tests/test-acceptance.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .Release.Name }}-connection-token-handler-tests
```
[the remark this answers is not in the document]
Comment (Maladie) [reply resources/application-connector/charts/connection-token-handler/templates/tests/test-acceptance.yaml]: Done

**Pull request merged**